apps/web/src/contexts/fullPageNftDetailModalListener/FullPageNftDetailModalListener.tsx

```diff
@@ -41,7 +41,7 @@ export default function FullPageNftDetailModalListener() {
     }
 
     const content = collectionId ? (
-      <Suspense fallback={<NftDetailPageFallback />}>
```

Comment:
instead of having to do `tokenId as string` here, on L39 let's update:
```ts
if (!username) {
  return
}
if (typeof tokenId !== 'string) {
  return
}
```

then you should safely be able to pass in `tokenId={tokenId}` without type casting